### PR TITLE
Make sure we handle a supplementary output list with no outputs in it

### PR DESCRIPTION
**Pull request by jrose-apple**
Ran across this while modifying command lines to build the standard library.

### CODE REVIEW

File: lib/Frontend/OutputFileMap.cpp [hunk 1]
```diff
@@ -111,10 +111,13 @@ void OutputFileMap::write(llvm::raw_ostream &os,
                           ArrayRef<StringRef> inputs) const {
   for (const auto input : inputs) {
     const TypeToPathMap *outputMap = getOutputMapForInput(input);
-    if (!outputMap)
-      continue;
     writeQuotedEscaped(os, input);
-    os << ":\n";
+    os << ":";
+    if (!outputMap) {
```
Comment (davidungar) [inline lib/Frontend/OutputFileMap.cpp]: How about adding a comment like "In case there are NO supplementary outputs"?

**Review by davidungar — APPROVED** [review]
Nice catch! Two comments:
1. Typo in the PR title "sure" is missing after "Make".
2. Would be nice to add a comment in OutputFileMap.

**Comment by jrose-apple** [comment]
@swift-ci Please smoke test

**Comment by davidungar** [comment]
Nice comments!

**Comment by jrose-apple** [comment]
@swift-ci Please smoke test macOS

**Comment by jrose-apple** [comment]
@swift-ci Please smoke test OS X

**Pull request merged**